Failed to load image: Invalid symbol 10, offset 19812.

### Check for existing issues

- [ ] Completed

### Describe the bug / provide steps to reproduce it

It shows  "Failed to load image: Invalid symbol 10, offset 19812." after  I run a python program with plotting function  using Zed REPL.  By the way,  the results are all correct except no pictures showed.
![截屏2024-08-16 11 05 30](https://github.com/user-attachments/assets/4f6f19c6-be79-4eb2-9d85-c66b15144f36)


### Environment

Mac OS 13.6 ,  Miniconda Python 3.9

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>
<!-- Click below this line and paste or drag-and-drop your log-->
2024-08-16T10:55:29.359898+08:00 [INFO] kernel: NOTE: When using the `ipython kernel` entry point, Ctrl-C will not work.
2024-08-16T10:55:29.360055+08:00 [INFO] kernel: 
2024-08-16T10:55:29.360175+08:00 [INFO] kernel: To exit, you will have to explicitly quit this process, by either sending
2024-08-16T10:55:29.360298+08:00 [INFO] kernel: "quit" from a client, or using Ctrl-\ in UNIX-like environments.
2024-08-16T10:55:29.360422+08:00 [INFO] kernel: 
2024-08-16T10:55:29.36054+08:00 [INFO] kernel: To read more about this, see https://github.com/ipython/ipython/issues/2049
2024-08-16T10:55:29.360664+08:00 [INFO] kernel: 
2024-08-16T10:55:29.360782+08:00 [INFO] kernel: 
2024-08-16T10:55:29.360901+08:00 [INFO] kernel: To connect another client to this kernel, use:
2024-08-16T10:55:29.361024+08:00 [INFO] kernel:     --existing kernel-zed-30064771095.json
<!-- Click above this line and paste or drag-and-drop your log--></pre></details>

@cthed2 Reported a similar error in dev (0.154.x) in #18086. Reopening here, hopefully we can come reproduce.


> The R code executes fine, I can't view images generated by ggplot2. Instead, I receive the following error message.

Failed to load image: Invalid symbol 10, offset 72

![image](https://github.com/user-attachments/assets/ba3ab308-6318-4e37-8d1e-1232699dc3b1)

```
2024-09-19T11:07:07.461096-05:00 [ERROR] Error { code: HRESULT(0x80070490), message: "No se ha encontrado el elemento." }
```

Settings:
```
"jupyter": {
  "kernel_selections": {
    "r": "ark"
  }
}
```

--- a/crates/repl/src/outputs/image.rs
+++ b/crates/repl/src/outputs/image.rs
@@ -16,7 +16,7 @@ pub struct ImageView {
 
 impl ImageView {
     pub fn from(base64_encoded_data: &str) -> Result<Self> {
-        let bytes = BASE64_STANDARD.decode(base64_encoded_data)?;
+        let bytes = BASE64_STANDARD.decode(base64_encoded_data.trim())?;
 
         let format = image::guess_format(&bytes)?;
         let mut data = image::load_from_memory_with_format(&bytes, format)?.into_rgba8();

